### ISSUE TITLE
flutter_driver from sdk is incompatible with get_it >=7.1.4

### ISSUE THREAD

**Issue by fullflash** [fluttercommunity/get_it#203]
unable to upgrade to latest get_it 7.1.4 package

```
Because every version of flutter_driver from sdk depends on async 2.6.1 and get_it >=7.1.4 depends on async ^2.7.0, flutter_driver from sdk is incompatible with get_it >=7.1.4.
So, because core depends on both get_it ^7.1.4 and flutter_driver any from sdk, version solving failed.
```

```
flutter doctor -v
[✓] Flutter (Channel stable, 2.2.2, on macOS 11.4 20F71 darwin-x64, locale
    en-TR)
    • Flutter version 2.2.2 at /Users/fersmart/SDK_all/flutter
    • Framework revision d79295af24 (4 days ago), 2021-06-11 08:56:01 -0700
    • Engine revision 91c9fc8fe0
    • Dart version 2.13.3

```

**Comment by escamoteur** [comment]
Hmm, that is not what I intended :-) On the other side 7.1.4 doesn't contain any change that will not affect most as its a bug fix that only is necessary if you use `registerSingletonWithDependencies` with `signalReady=true`.

Please let me know if its ok for you to stay on 7.1.3 then

**Comment by escamoteur** [comment]
@csells I don't know who is responsible for flutte_driver that it doesn't use the latest published version of the async package

**Comment by escamoteur** [comment]
fixed with V7.2.0
